app/screens/AppNavigator/screens/Settings/SettingsScreen.tsx

```diff
@@ -34,12 +34,13 @@ export function SettingsScreen ({ navigation }: Props): JSX.Element {
     }
 
     const auth: Authentication<string[]> = {
-      message: translate('screens/Settings', 'To continue downloading your recovery words, we need you to enter your passcode.'),
       consume: async passphrase => await MnemonicStorage.get(passphrase),
       onAuthenticated: async (words) => {
         navigation.navigate({ name: 'RecoveryWordsScreen', params: { words }, merge: true })
       },
-      onError: e => Logging.error(e)
+      onError: e => Logging.error(e),
+      message: translate('screens/Settings', 'To continue downloading your recovery words, we need you to enter your passcode.'),
+      loading: translate('screens/Settings', 'Decrypting recovering words...')
```

Comment:
Don't use "decrypting" as it might alarm security that their wallet will be unsafe. Let's keep it generic like Loading...